Makefile.dep

```diff
@@ -186,6 +186,10 @@ ifneq (,$(filter ng_slip,$(USEMODULE)))
   USEMODULE += ng_netbase
 endif
 
+ifneq (,$(filter ipv6_addr,$(USEMODULE)))
+  USEMODULE += ipv4_addr
```

Comment:
Its used for instance for encapsulated IPv4 addresses [1]

[1] https://github.com/RIOT-OS/RIOT/pull/3612/files#diff-23b03f9f8cb9719704da8643a577382aR107